src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -5,7 +5,17 @@ import { SystemIntakeReview } from 'components/SystemIntakeReview';
 import MainContent from 'components/MainContent';
 import Header from 'components/Header';
 import { AppState } from 'reducers/rootReducer';
-import { fetchSystemIntake } from 'types/routines';
+import { fetchSystemIntake, reviewSystemIntake } from 'types/routines';
```

Comment:
I'm still not sure how, but we should backlog a task on how we manage out imports, whether it's ESLint or Prettier. Things are getting unorganized.

For example:
NPM Imports
views
components,
etc,
etc,
relative imports

Reply:
Created a ticket for this https://jiraent.cms.gov/browse/EASI-691

Reply:
Thanks!